### DISCUSSION TITLE
Do I need to set up a Usenet Blackhole client to process the Tidal/deezer imports

### DISCUSSION THREAD

**Discussion by Darkcloud1987** [RandomNinjaAtk/arr-scripts#273] [Q&A] [answered]
So Tidal/Deezer downloads where staying in the completed folder that I chose in the config.  I also regulary got an "nzbfolder does not exist" error in the logs. Now I did set up a black hole pointing to an empty nzb directory and the completed folder. The downloads there are now in the queue and the error message does not pop up anymore. So is this the right way or are the downloads actualy getting picked up by a background script that might now colide with the black hole?

**Comment by RandomNinjaAtk** [comment, marked as answer]
Yes, you do. However, it should have set it up for you automatically...

**Comment by Darkcloud1987** [comment]
strange, I was thinking that it should have set it up and the confusion came because it wasn't for some reason.

**Reply by RandomNinjaAtk** [reply]
If it did not autoconfigure it for you, then there might be a bug... because it should have...

**Comment by Darkcloud1987** [comment]
One thing is that I did create the folders through the synology Container Manager and restarted the container from there. I also opened the console there to get the link for the Tidal client login but could not close the client after login. So if the script runs after that this might be the issue but the downloading itself works.

Another thing is, that the container hung through restart so I had to restart the whole diskstation. At that point the settings where still on only deezer. Sso either it didn't go through there or the heavy load afterwards prevented it.
